scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/ExtendedSchedulerPolicy.java

```diff
@@ -64,69 +63,64 @@
  */
 public class ExtendedSchedulerPolicy extends DefaultPolicy {
 
-    private static final Logger logger = Logger.getLogger(ExtendedSchedulerPolicy.class);
```

Comment:
It would be nice to keep existing formatting or to use existing formatting profile. If remember, _4_ spaces is the default for indentation please do not replace by tabulation.